aiida_workgraph/orm/general_data.py

```diff
@@ -16,12 +19,14 @@ def value(self):
 
 
 class GeneralData(orm.Data):
-    """`Data to represent a pickled value."""
+    """Data to represent a pickled value using cloudpickle."""
+
+    FILENAME = "value.pkl"  # Class attribute to store the filename
```

Comment:
So 6his is more like a`PickleObject` data type righ?

Reply:
There is no `PickleObject` data type, right? The pickled data is just binary format data, and then we store the binary data in the repository.

Reply:
We can change the name `GeneralData` to `PickledData`, or `AnyData`.